department-of-veterans-affairs/vets-website
# Update POW Status page

## Summary
Add custom service period review on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the 'P. O. W. Status' page in the 'Military history' chapter
3. Verify that the screen reader reads out the title 'P. O. W. Status' instead of the word 'POW' and 'Status'

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75888)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_military_pow (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1dcdabdc-8609-4ffc-9eb5-498241a31450)|![localhost_3001_pension_application_527EZ_military_pow](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ad107579-a3ea-44a3-9a3e-4c81255a5209)





## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
